app/src/main/java/net/squanchy/navigation/firststart/FirstStartWithNoNetworkActivity.java

```diff
@@ -33,6 +33,7 @@
 import io.reactivex.functions.Consumer;
 import timber.log.Timber;
 
+@SuppressWarnings("PMD.ExcessiveImports")           // Might use some refactoring later on to extract some collaborator
```

Comment:
same here, I'd rather not risk sneaking in new issues because of non-zero thresholds